### PR TITLE
DE-6867: Adding example for running PyFlink on Decodable

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: pyflink-decodable/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+By deploying your PyFlink jobs as [https://docs.decodable.co/pipelines/create-pipelines-using-your-own-apache-flink-jobs.html](Custom Pipelines) onto Decodable, you can solely focus on implementing your job,
+while leaving all the aspects of running the job, like provisioning Flink clusters and the underlying hardware,
+keeping them secure and up-to-date, scaling them, monitoring and observing them, to the fully-managed Decodable platform.
+
+## Prerequisites
+
+You'll need the following things in place to run this example:
+
+* A free Decodable account ([https://app.decodable.co/-/accounts/create](sign up))
+* The [Decodable CLI](https://docs.decodable.co/cli.html)
```
Comment (enthal) [inline pyflink-decodable/README.md]: Happened to see this: in MD links it goes `[text](url)` (so as you have it in the second line here, but not the first).  There's at least one other place in this file with this mistake. 😇
```suggestion
* A free Decodable account ([sign up](https://app.decodable.co/-/accounts/create))
* The [Decodable CLI](https://docs.decodable.co/cli.html)
```

Comment (gunnarmorling) [reply pyflink-decodable/README.md]: Argh, it happens _all the time_! Thx for fixing!

File: pyflink-decodable/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Getting Started With PyFlink On Decodable
+
+This example project shows how to run PyFlink jobs on Decodable.
+
+By deploying your PyFlink jobs as [https://docs.decodable.co/pipelines/create-pipelines-using-your-own-apache-flink-jobs.html](Custom Pipelines) onto Decodable, you can solely focus on implementing your job,
```
Comment (enthal) [inline pyflink-decodable/README.md]:
```suggestion
By deploying your PyFlink jobs as [Custom Pipelines](https://docs.decodable.co/pipelines/create-pipelines-using-your-own-apache-flink-jobs.html) onto Decodable, you can solely focus on implementing your job,
```

File: pyflink-decodable/Makefile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+TARGET = target
+LIBS = $(TARGET)/libs
+PYTHON_LIBS = $(TARGET)/python-libs
+
+PY = python3
+VENV = $(TARGET)/venv
+BIN=$(VENV)/bin
+
+build: $(TARGET) $(LIBS)/flink-sql-connector-kafka-3.0.2-1.18.jar $(LIBS)/flink-python-1.18.1.jar $(TARGET)/pyflink-job.zip $(VENV)
```
Comment (jbreeden) [inline pyflink-decodable/Makefile:11]: Is this possibly out of date already? I attempted to build this and hit a 404 on the jar:

```
~/decodable/repos/examples/pyflink-decodable DE-6867
❯ make
mkdir target
mkdir -p target/libs
wget -N -P target/libs https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/3.0.2-1.18/flink-sql-connector-kafka-3.0.2-1.18.jar
make: wget: No such file or directory
make: *** [target/libs/flink-sql-connector-kafka-3.0.2-1.18.jar] Error 1
```

Comment (jbreeden) [reply pyflink-decodable/Makefile:11]: My bad, clearly I just like `wget`

**Review by rmoff — APPROVED** [review]

**Pull request merged**